weather_mv/loader_pipeline/sinks_test.py

```diff
@@ -38,6 +43,25 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def generate_dataset() -> str:
```

Comment:
Let's name this something like "write_netcdf".

Reply:
Looking at how this is used: what do you think about making this into a contextmanager? The difference, besides adding a decorator, is that we'd `yield` from the tempfile `with` statement. The benefit is that we'd get cleanup "for free". Would this negatively affect your memory test?

Reply:
Appropriate Changes were done.